agents/sources.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const opentrialsApi = require('../config').opentrialsApi;
+
+function listSources() {
+  return opentrialsApi
+    .then((client) => client.sources.list())
+    .then((response) => response.obj);
+}
+
+module.exports = {
+  list: listSources,
```

Comment:
Same as for the API PR, I would go with `getSources` as `operationId`, then export it as it is, using the new ES6 syntax ;)